pubspec.yaml

```diff
@@ -3,10 +3,10 @@
 # BSD-style license that can be found in the LICENSE file.
 
 name: usage
-version: 3.4.2-dev
-description: A Google Analytics wrapper for both command-line, web, and Flutter apps.
+version: 3.4.2
+description: >-
+  A Google Analytics wrapper for both command-line, web, and Flutter apps.
```

Comment:
The "both" is awkward here. Should we rephrase it?

Reply:
ack